x-pack/test/functional_with_es_ssl/apps/triggers_actions_ui/rules_list/maintenance_window_banner.ts

```diff
@@ -0,0 +1,50 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { FtrProviderContext } from '../../../ftr_provider_context';
+import { ObjectRemover } from '../../../lib/object_remover';
+import { generateUniqueKey } from '../../../lib/get_test_data';
+import { createMaintenanceWindow, createObjectRemover } from '../maintenance_windows/utils';
+
+export default ({ getPageObjects, getService }: FtrProviderContext) => {
```

Comment:
I believe we might achieve the desired test confidence using a Jest test, which could run faster and might reduce flakiness. What do you think about considering a Jest test for this instead of an e2e test?

Reply:
@jcger eb5c988746cc06afb20c309b2224849821a2c2e8

Reply:
I think we can just do it like that 7a89261997233f667ad83cb3de5745d1cd92cab1 without too much complication and maybe avoid e2e test. what do you think?